core/Cargo.toml

```diff
@@ -47,7 +47,7 @@ rust-version = "1.75"
 version = "0.50.1"
 
 [features]
-default = ["reqwest/rustls-tls", "executors-tokio", "services-memory"]
+default = ["reqwest/rustls-tls", "executors-tokio", "services-memory" ,"services-memcached"]
```

Comment:
Please do not add `services-memcached` as the default. We want users to enable only the memory service by default.